rsmanage/rsmanager.py

```diff
@@ -146,24 +146,28 @@ def addcourse(config, course_name, basecourse, start_date, python3, login_requir
         if not course_name:
             course_name = click.prompt("Course Name")
         if not python3:
-            python3 = 'T' if click.confirm("Use Python3 style syntax?", default=True) else 'F'
+            python3 = 'T' if click.confirm("Use Python3 style syntax?", default='T') else 'F'
+        else:
+            python3 = 'T'
```

Comment:
Need 'T' or 'F' in DB insert, not True or missing. Concern: is 'T' and 'F' for booleans specifically a PSQL thing, or is that generic for all SQL?

Reply:
You don't want the else there -- the way this is meant to work is that python3 either be defined by a command line parameter, or the user will be prompted for it.

Postgresql conforms to the SQL standard in how it implements the boolean data type.

Reply:
Which else do you think is unneeded? I think line 150-151 are needed because when python3 is set as a command flag, its value is True rather 'T'.

Reply:
Ahh I think you are right.... I

f the --python3 option is omitted then the python3 parameter will be None which causes click.confirm() to ask the user.  It can return true as the conditional assignment uses 'T' so you don't need to change the default  to 'T' but that probably doesn't matter.

But if the --python3 flag is set, it needs to be changed to 'T' rather than True otherwise the insert will fail.

I swear I testing this subcommand and it worked... but looking at the execute statement I can't see how.

I suppose I really should clean this up by using the declarative model of SQLAlchemy and having using the constructor to add the row.